src/Blocks/BlockHandler.cpp

```diff
@@ -636,9 +636,9 @@ void Temporary::RegisterAllBlockHandlers(BlockTypeRegistry & aRegistry)
 {
 	static struct
 	{
-		const char * newBlockTypeName;
-		BLOCKTYPE oldBlockType;
-	} regs[] =
+		const char * NewBlockTypeName;
```

Comment:
Members of local structs should be lowerCamelCase I think, unless the style guide says otherwise?

Reply:
Members should be `m_NewBlockTypeName` ideally, or left unchanged.

Reply:
Since this PR doesn't change the code logic at all, only cosmetics, this change shouldn't be here.